torch_xla/csrc/aten_xla_bridge.h

```diff
@@ -14,7 +15,10 @@ namespace bridge {
 
 // Extracts the XLATensor out of our version of at::Tensor. Throws an exception
 // if tensor is not an XLA tensor.
-XLATensor& GetXlaTensor(const at::Tensor& tensor);
+XLATensor GetXlaTensor(const at::Tensor& tensor);
+
+// Fetches the underline XLATensor object if tensor is an XLA tensor.
+c10::optional<XLATensor> TryGetXlaTensor(const at::Tensor& tensor);
```

Comment:
Why is this in the header? It's a helper only used in `torch_xla/csrc/aten_xla_bridge.cpp` as far as I can tell.

Reply:
I can see other parts of the codebase wanting a do-this-if-xla-tensor-else-...

Reply:
It's very easy to move something out of an anonymous namespace and add its signature to a header. All else being equal, smaller interfaces are nicer.

Reply:
Alrighty.